qiita_db/reference.py

```diff
@@ -23,6 +79,7 @@ class Reference(QiitaObject):
     sequence_fp
     taxonomy_fp
     tree_fp
+    sortmerna_indexed_db
```

Comment:
@wasade I thought about that, too, but this class is also assuming that it will have a taxonomy and tree, so I figured if it is assumed to have those things, it can be assumed to have a sortmerna indexed db, too, right? In any case, if it isn't passed, a warning is raised (not an error), but maybe it shouldn't even be a warning?